### DISCUSSION TITLE
the difference among ebsd properties (bc, iq, ci...)

### DISCUSSION THREAD

**Discussion by pxyg** [mtex-toolbox/mtex#1242] [Ask Anything] [answered]
Dear all, 

I feel a little confused about the ebsd properties. When I process my data with "ctf" file, the ebsd properties include bands, bc, bs, error... However,  I saw in some codes. the ebsd properties include ci, fit, iq... It seems that in this case the original file is "ang". 

My question is, are these properties with different names have the exact same meaning and values. For example, when I want to plot an IQ map, I would use plot (ebsd, ebsd.bc), is the property "bc" from the "ctf"exactly as the same as the "iq" from the "ang" file. Where can I find the relationship between these different properties. If I want to plot a "grain average image quality" map, can I directly use the values of the property "bc" or some other properties?

Thanks!

**Comment by kilir** [comment, marked as answer]
Hi,
the values are vendor specific and not mostly not comparable. Some  give quite similar impressions (e.g. iq and bc). As far as I know, none of the vendors provides a chart of how their properties relate to those of others.

Cheers,
Rüdiger

**Reply by pxyg** [reply]
Hello Rüdiger, 

do you maybe know how to transfer a ctf file into an ang file, namely transfer the properties bs,bc error... into ci,fit, iq...?

**Reply by filippeof** [reply]
They might have simillar meaning (how good the acquisition and indexing of an EBSP was) but certainly different values.

- For a description of band slope (bs) and band contrast (bc) from Oxford/HKL, see section 21.2 of the Channel 5 manual (available here for example: https://aarc.ua.edu/wp-content/uploads/docs/JEOL-7000F-Oxford_Channel_5_User_Manual.pdf)

- For image quality (iq) from EDAX, see:

Wright, Stuart I., and Brent L. Adams. “Automatic Analysis of Electron Backscatter Diffraction Patterns.” Metallurgical Transactions A 23, no. 3 (March 1, 1992): 759–67. https://doi.org/10.1007/BF02675553.
or:
Wright, Stuart I., and Matthew M. Nowell. “EBSD Image Quality Mapping.” Microscopy and Microanalysis 12, no. 1 (February 2006): 72–84. https://doi.org/10.1017/S1431927606060090.

- pattern quality (pq?) of Bruker:

https://www.bruker.com/en/products-and-solutions/elemental-analyzers/eds-wds-ebsd-SEM-Micro-XRF/software-esprit-family/data-presentation-options-postprocessing.html

cheers,
Filippe

**Reply by pxyg** [reply]
Thanks a lot for the detailed reply, Filippe!
